src/devices/Mcp25xxx/Register/MessageReceive/RxBxDlc.cs

```diff
@@ -66,12 +76,12 @@ private Address GetAddress()
         {
             switch (RxBufferNumber)
```

Comment:
kinda feels all these switches (this and similar for other enum) will be equivalent to: `(Address)(1 << RxBufferNumber)` minus the range check

Reply:
I'm not sure I follow.  Using RxBxDlc and if RxBufferNumber = 1, then...

(Address)(1 << RxBufferNumber) = (Address)(1 << 1) =  (Address)2 = RxF0Eid8

This is wrong as RxBufferNumber represents Address.RxB1Dlc for this register.

Reply:
right